Gopkg.toml

```diff
@@ -29,11 +29,11 @@ required = ["github.com/3rf/mongo-lint"]
 
 [[constraint]]
   name = "github.com/mongodb/mongo-tools-common"
-  branch = "TOOLS-2268"
+  version = "v1.0.1"
```

Comment:
This should be "^v1.0.1" but master already has that so just don't lose it in the conflict resolution.  (No need to fix now as part of this PR.)